.gitignore

```diff
@@ -1,8 +1,13 @@
 # Documentation
 docs/build/*
+docs/examples
 
 *.py[cod]
 
+# Exmaples
+# examples 40_advanced generate a tmp_folder
+examples/40_advanced/tmp_folder
```

Comment:
Which one does that? There should be no example doing that.

Reply:
Example `example_debug_logging.py` [here](https://github.com/automl/auto-sklearn/blob/6f1e5c33bf3cec21a80bd88a531f6019e59e9586/examples/40_advanced/example_debug_logging.py#L90)